uaclient/entitlements/fips.py

```diff
@@ -155,7 +211,11 @@ def _allow_fips_on_cloud_instance(
             if series in ("bionic", "focal"):
                 return True
 
-            return bool("ubuntu-gcp-fips" in super().packages)
+            super_packages = super().packages
+            if super_packages is not None:
+                return bool("ubuntu-gcp-fips" in super_packages)
```

Comment:
If `packages` return an empty list, we would not need that logic.
But again, I can still be missing the rationale for that change